### ISSUE TITLE
Meteor needs new package and maintainer

### ISSUE THREAD

**Issue by rosskevin** [mdbootstrap/mdb-ui-kit#930]
We would like to partner with another party that would like to maintain/publish a _separate package_ for meteor.  The maintainers of BMD do not use meteor and therefore feel inadequate to assist with any of the questions, and meteor questions appear more frequently in this issue tracker than most.

As of now, I have made the meteor package unavailable, the following **will no longer work** `meteor add fezvrasta:bootstrap-material-design`

If you would like to maintain a proper meteor packaging of Material Design for Bootstrap, please publish your package and we will happy to link to you.

/cc @FezVrasta

**Comment by Billybobbonnet** [comment]
With the release of Meteor 1.3, it seems that packages are very likely going to be replaced by npm in the meteor ecosystem. My bet is that it might happen before 2017. For Meteor projects, the way forward would just be to use npm imho. That would have been my first choice if it wasn't for a [pre-release bug with dynamic imports](https://forums.meteor.com/t/how-to-use-npm-materialize-css-package-with-material-ui-in-meteor-1-3/18272/4) I had with materialize and I expected to have with BMD.

Bottom line: maybe a simple tutorial, or even a paragraph in the readme will be necessary to use BMD in Meteor, since npm is now "fully" supported. I would be glad to contribute if I get it done.

By the way, I just discovered your package (& @FezVrasta's) today but I guess that many people (including me) want to thank you for the time spent on it so far, both you and every contributors. Thanks for making things happens :-)

**Comment by rosskevin** [comment]
@Billybobbonnet we are happy to include your doc.  Feel free to just start a wiki page here (mark it a work in progress), when you are done we'll link it from the README.  Thanks for the information!

**Comment by tsega** [comment]
@rosskevin, sad to hear that official Meteor support has been discontinued. I actually used BMD for almost all my projects nowadays and I would be more than happy to create a package for it. Actually, it's not only a matter of preference but of necessity!

@Billybobbonnet the official core Meteor release is still at `1.12.1` and I do use the stable version to work on my apps. It would be interesting to see how things change with the "full" NPM support but until then I think we still need a normal Meteor package.

**Comment by tsega** [comment]
@Billybobbonnet Meteor 1.3 has been officially released :balloon:  and now NPM packages are first class citizens of the Meteor ecosystem, kudos :+1: 

@rosskevin this definitely means there is no need to maintain a separate atmosphere package and I'm sure most pre 1.3 app would upgrade pretty soon!

**Comment by rosskevin** [comment]
Yay, I'll pull out the meteor stuff.
